src/extension/_locales/en/messages.json

```diff
@@ -478,7 +478,7 @@
       "message": "Published: "
     },
     "reload": {
```

Comment:
Should we change the key too?

Reply:
Changing just the key would be fine with me, but I think I wouldn't change the plugin name.